# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../screen/glibc23.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./extern.h.orig	2003-03-12 17:57:42.000000000 +0100
+++ ./extern.h	2003-03-12 17:58:09.000000000 +0100
@@ -344,9 +344,9 @@
 #if defined(POSIX) || defined(hpux)
 extern void (*xsignal __P((int, void (*)SIGPROTOARG))) __P(SIGPROTOARG);
 #endif
-#ifndef HAVE_RENAME
-extern int   rename __P((char *, char *));
-#endif
+// #ifndef HAVE_RENAME
+// extern int   rename __P((char *, char *));
+// #endif
 #if defined(HAVE_SETEUID) || defined(HAVE_SETREUID)
 extern void  xseteuid  __P((int));
 extern void  xsetegid  __P((int));
--- ./os.h.orig	2003-03-12 18:00:22.000000000 +0100
+++ ./os.h	2003-03-12 18:01:00.000000000 +0100
@@ -172,9 +172,9 @@
 #ifndef HAVE_UTIMES
 # define utimes utime
 #endif
-#ifndef HAVE_VSNPRINTF
-# define vsnprintf xvsnprintf
-#endif
+// #ifndef HAVE_VSNPRINTF
+// # define vsnprintf xvsnprintf
+// #endif
 
 #ifdef BUILTIN_TELNET
 # include <netinet/in.h>
--- ./misc.c.orig	2003-03-12 17:58:44.000000000 +0100
+++ ./misc.c	2003-03-12 17:59:45.000000000 +0100
@@ -76,7 +76,7 @@
 int err;
 {
   extern int sys_nerr;
-  extern char *sys_errlist[];
+//extern char *sys_errlist[];
 
   static char er[20];
   if (err > 0 && err < sys_nerr)
@@ -423,6 +423,7 @@
 #endif
 }
 
+#if 0
 #ifndef HAVE_RENAME
 int
 rename (old, new)
@@ -434,6 +435,7 @@
   return unlink(old);
 }
 #endif
+#endif
 
 
 int
